app/townsquare/templates/townsquare/index.html

```diff
@@ -89,7 +75,19 @@
               </ul>
             </div>
           </div>
-          <div class="row mt-4 townsquare_header">
+          <div class="row {% if user.user.is_authenticated %} mt-4 {% else %} mt-2 {% endif %}townsquare_header">
```

Comment:
typo here: `user.user.is_authenticated` -> `user.is_authenticated`. Does this still work somehow?

Reply:
No, it doesn't display the correct margin, my bad. :sweat_smile: Thanks!